test/Microsoft.AspNet.Mvc.TagHelpers.Test/SelectTagHelperTest.cs

```diff
@@ -356,7 +369,7 @@ public class SelectTagHelperTest
                     null,         // optionLabel
                     string.Empty, // name
                     expectedItems,
-                    expectedAllowMultiple,
+                    false,        // allowMultiple
```

Comment:
yes, existing test `ProcessAsync_CallsGeneratorWithExpectedValues_ItemsAndMultiple()` demonstrates the important thing -- calculating `allowMultiple` based on the type.